vuepress/docs/next/tutorials/ecr/github-actions-workflow.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebarDepth: 2
+---
+
+# Add a GitHub Actions CI Workflow
+
+## Overview
+This tutorial shows how to use the JHipster `ci-cd` sub-generator to quickly create a basic GitHub Actions Workflow for the microservices and micro frontends (MFEs) generated by the Entando Component Generator.
+
+## Prerequisites
+* An existing project with a GitHub repository. See the [Entando Component Generator](../backend-developers/generate-microservices-and-micro-frontends.md#overview) to create a new one.
+* [GitHub Actions](https://github.com/features/actions). These are enabled by default without additional configuration. Please note public repositories currently qualify for unlimited usage but private repositories can encounter usage restrictions.
```

Comment:
'Please note that public repositories currently...' is 'that' needed here?

Reply:
@jyunmitch I don't see a that in the current version? So, I'd say nope.